lib/miq_automation_engine/service_models/miq_ae_service_vm.rb

```diff
@@ -1,5 +1,7 @@
 module MiqAeMethodService
   class MiqAeServiceVm < MiqAeServiceVmOrTemplate
+    expose :set_remote_console_url
```

Comment:
I think you only want the Automate method to take a single URL parameter, but you need to pass URL and user parameters to the active_record model

In that case do not use `expose`, define the method like this:
```ruby
def set_remote_console_url(url)
  object_send(:set_remote_console_url, url, DRb.front.workspace.ae_user.id)
end
```

@mkanoor Is this the best way to get the user id?